src/gui/clips/AutomationClipView.cpp

```diff
@@ -486,5 +486,55 @@ void AutomationClipView::scaleTimemapToFit( float oldMin, float oldMax )
 	m_clip->generateTangents();
 }
 
+//! Split this Clip.
+/**
+	param pos the position of the split, relative to the start of the clip
+*/
```

Comment:
```suggestion
```
Rather than adding duplicate comments to the source files, I would update the comment for `splitClip` in ClipView.h.

You could use a single multi-line doxygen comment like this:
```c++
/**
 * Split this Clip into two clips
 * @param pos the position of the split, relative to the start of the clip
 * @return true iff the clip could be split
 */
```